frb_codegen/src/generator/rust/mod.rs

```diff
@@ -554,6 +583,33 @@ pub fn generate_list_allocate_func(
     )
 }
 
+pub fn generate_list_deallocate_func(
```

Comment:
Not sure, shall we put *list* dealloc func here in *mod*, or is it possible to be in list-related files?

Reply:
i placed `generate_list_deallocate_func` under `generate_list_allocate_func`

Reply:
I see. So it is a tech debt.

Reply:
validation delegated to rust wire2api